lib/landable/version.rb

```diff
@@ -1,8 +1,8 @@
 module Landable
   module VERSION
     MAJOR = 1
-    MINOR = 12
-    PATCH = 1
+    MINOR = 13
```

Comment:
Dont do version bumps in PRs.  Do update the changelog

When we are ready to do a release to ruby gems we do all in one commit to master..